src/applications/mhv/medications/util/helpers.js

```diff
@@ -48,3 +49,20 @@ export const validateField = (fieldValue, date = false) => {
   }
   return 'None noted';
 };
+
+/**
+ * @param {String} fieldValue value that is being validated
+ */
+export const getImageUri = fieldValue => {
```

Comment:
I'd like to see unit tests for this function to confirm that we get the expected behavior from it.

Reply:
I will add unit test in another ticket because it should be merged as soon as possible.
Vitalli wants to avoid conflict with my update.